include/jemalloc/internal/spin_delay_arm.h

```diff
@@ -0,0 +1,23 @@
+#include "jemalloc/internal/jemalloc_preamble.h"
+#include <stdatomic.h>
+
+/* Global variable to track SB support, declared as extern to be defined in one TU */
+extern _Atomic int arm_has_sb_instruction;
```

Comment:
Marking this as atomic may regress the caller. This does not need to be atomic, though, so it should be changed to a normal int.

If the caller uses default loads on an atomic variable, these loads have memory-order-seq-cst. If the caller uses explicit atomic loads with memory-order-relaxed, it is still a regression because many compilers will not hoist the load out of a loop.